packages/date-time-utilities/src/dateFormatting/dateFormatting.types.ts

```diff
@@ -23,3 +23,16 @@ export interface IDateGridStrings {
    */
   shortDays: string[];
 }
+
+export interface IDateFormatting {
+  /**
+   * Format the date according to specified function.
+   * Intended use case is localization.
+   */
+  format: (date: Date) => string;
```

Comment:
What is the actual relation with the formatting strings above? Does the method override? Where is the intended usage? Source is [the design document](https://github.com/microsoft/fluentui/blob/master/specs/Datepicker.md).

Reply:
Input, aria label pro konkretni dny.